src/actions/ServerApiActions.js

```diff
@@ -4,23 +4,50 @@ import {
   SET_IS_TOKEN_RECEIVED,
   SET_TOKEN_EXPIRATION_TIME
 } from '../actionTypes';
+import { API_AUTH_PASS, API_AUTH_USERNAME } from '../constants/constants';
+import { weatherAPI } from '../services/dataService';
 
 export const setIsTokenReceived = isTokenReceived => ({
   type: SET_IS_TOKEN_RECEIVED,
-  payload: { isTokenReceived }
+  payload: {
+    isTokenReceived
+  }
 });
 
 export const setTokenExpirationTime = tokenExpirationTime => ({
   type: SET_TOKEN_EXPIRATION_TIME,
-  payload: { tokenExpirationTime }
+  payload: {
+    tokenExpirationTime
+  }
 });
 
 export const setIsFetchingInProgress = isFetchingInProgress => ({
   type: SET_IS_FETCHING_IN_PROGRESS,
-  payload: { isFetchingInProgress }
+  payload: {
+    isFetchingInProgress
+  }
 });
 
-export const setIsFetchingError = fetchingError => ({
+export const setFetchingError = fetchingError => ({
   type: SET_FETCHING_ERROR,
-  payload: { fetchingError }
+  payload: {
+    fetchingError
+  }
 });
+
+export const getToken = () => dispatch => {
```

Comment:
OPTIONAL: you could omit additional closure: 
`export const getToken = dispatch => {`

Reply:
I see. I prefer to leave this closure for consistency as we discussed in chat.